apps/emqx_durable_storage/src/proto/emqx_ds_proto_v4.erl

```diff
@@ -179,8 +179,7 @@ make_delete_iterator(Node, DB, Shard, Stream, TopicFilter, StartTime) ->
     | {ok, end_of_stream}
     | {error, _}.
 delete_next(Node, DB, Shard, Iter, Selector, BatchSize) ->
-    emqx_rpc:call(
-        Shard,
+    erpc:call(
```

Comment:
Using janky gen_rpc is only justified when forwarding large binaries, however this is not the case here.